model/riscv_sys_regs.sail

```diff
@@ -569,8 +569,7 @@ register minstret : bits(64)
 register minstret_increment : bool
```

Comment:
Missed the comment just above here, otherwise looks fine; feel free to merge with that fixed as far as I'm concerned

Reply:
I can't wait until we have autoformatting in place.